Apps/Playground/0.70/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+// prettier-ignore
```

Comment:
Should we use a common tsconfig for all RN versions?

Reply:
Might be possible. Not part of this PR.